docusaurus/docs/reactnative/core-components/channel.mdx

```diff
@@ -290,6 +291,10 @@ This is often the header height.
 
 <CopyMessage />
 
+### deletedMessagesVisibiltyType
```

Comment:
This should also be added to all the other docs, where its user e.g., MessagesContext, Message, MessageContent etc. Please review all the components, contexts where it needs to be added in docs.